cmd/swctl/app/cmd_dependency.go

```diff
@@ -747,3 +755,42 @@ func isUserRoot() (bool, error) {
 	return true, nil
 
 }
+func postInstall(cli Cli) []error {
+	var errorGroup []error
+	sudoName, err := logname(cli)
+	fmt.Fprintln(cli.Out(), sudoName)
+	// handling case when linux has no login name (e.g. container)
+	if err == nil {
+		commands := []string{
+			"usermod -aG docker " + sudoName, // add user do docker group
+		}
+
+		for _, command := range commands {
+
+			out, stderr, err := cli.Exec(GetSudoPrefix(cli)+"bash -c", []string{command}, false)
+
+			if stderr != "" {
+				errorGroup = append(errorGroup, errors.New(command+": "+stderr))
+			}
+			if err != nil {
+				errorGroup = append(errorGroup, errors.New(err.Error()+"("+command+")"))
+			}
+			color.Fprintln(cli.Out(), out)
```

Comment:
Again, debug logging? If not then add what is this output for, i.e. color.Fprinf(cli.Out(), "adding user to docker user group...\n%s", out)